vcs/git.txt

```diff
@@ -0,0 +1,101 @@
+| Store User       | $ git config credential.helper store                      |
+| Credentials      | # credentials saved in ~/.git-credentials as clear text   |
+|                  |                                                           |
+| Clone Repository | $ git clone GIT_URL                                       |
+|                  |                                                           |
+| Clone Branch     | $ git clone GIT_URL -b BRANCH_NAME                        |
+|                  |                                                           |
+| Repo. Status     | $ git status                                              |
+|                  |                                                           |
+| Add/Update       | $ git add FILE_OR_PATH                                    |
+| Project Files    |                                                           |
+|                  |                                                           |
+| Remove           | $ git add FILE_OR_PATH                                    |
+| Project Files    |                                                           |
+|                  |                                                           |
+| Commit           | $ git commit                                              |
+|                  | # records changes to the repository                       |
+|                  |                                                           |
+| Push             | $ git push                                                |
+|                  | # updates remote refs along with associated objects       |
+|                  |                                                           |
+| Pull             | $ git pull                                                |
+|                  | # fetch and integrate with another repository/branch      |
+|                  |                                                           |
+| Fetch            | $ git fetch                                               |
+|                  | # download objects and refs from another repository       |
+|                  |                                                           |
+| Merge            | $ git merge                                               |
+|                  | # join two or more development histories together         |
+|                  |                                                           |
+| Rebase           | $ git rebase                                              |
+|                  | # reapply commits on top of another base tip              |
+|                  |                                                           |
++------------------------------------------------------------------------------+
+
+
+Example 1: Staying Up-to-Date [1]
+________________________________________________________________________________
+
+In a standard setup, you generally have an origin and an upstream remote — the 
```

Comment:
`—` -> `-` 

I'd love to use this fancier stuff though mono-space font support outside of ASCII is surprisingly horrible on a lot of mobile devices! Even box drawing characters are unusable for this reason. We could of course ignore this possibility and allow the odd breakage. Will think about it.